packages/mgt-spfx/config/package-solution.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/spfx-build/package-solution.schema.json",
+  "solution": {
+    "name": "mgt-spfx-client-side-solution",
+    "id": "e342d2ab-fc3f-45f6-93a5-268fd8632211",
+    "version": "1.0.0.0",
```

Comment:
same with this version, should this be updated to be the same version as the package?

Reply:
Yes. The caveat here is, that it needs to be 4-part rather than just semver, so if we can append the extra `.0` automatically, it would work.

Reply:
Do we need to update this manually each time the version changes, or will it get automatically updated?

Reply:
Right now we'd need to update it manually, unless there is a way for us to tie into the current CI/CD setup of MGT. We could keep the last `.0` static and align the first three digits with the package.json version if that's possible

Reply:
We have a [setVersion](https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/main/scripts/setVersion.js) script that sets the version of all the packages - it could be updated to also update this version. For preview versions though, we append the short commit hash (ex: 2.1.0-ae84ed7), I assume that would not work in this case?

Reply:
I don't think so. As far as I can tell, the accepted format follows the assembly versioning scheme with 4 numbers.

Reply:
Got it. It sounds like we would have to keep the format then. Is there concern with different versions of the mgt-spfx package having the same version of the package-solution.json? Would that cause issues for upgrading from one package to another (going from one preview version to another)? Or would we have to leverage the fourth number somehow?

Reply:
You can have only one version of the solution installed in the app catalog and preview versions that would require the 4th number, would only affect beta testers, so I don't think we need to worry about it.